Scrabble.java

```diff
@@ -85,10 +114,23 @@ public static void playHand(String hand) {
 			// non-whitespace characters. Whitespace is either space characters, or  
 			// end-of-line characters.
 			String input = in.readString();
-			//// Replace the following break statement with code
-			//// that completes the hand playing loop
-			break;
+
+			if (input.equals(".")) {
+				break;
+			}
+
+			input = input.toLowerCase(); // normalizing input to lowercase letters
+
+			if (isWordInDictionary(input) && MyString.subsetOf(input, hand)) {

```

Comment:
Split to 2 different condition and handle the not values. 
for better practice

Reply:
Thank you very much